csharp/Readme.md

```diff
@@ -7,28 +7,32 @@ products:
 description: "Azure SQL - Getting Started Samples"
 ---
 
-# Developing applications with C# and Azure SQL 
+# Developing applications with C# and Azure SQL
 
-This repository contains a tutorial that will guide you through the creation of a simple solution using [C#](https://docs.microsoft.com/en-us/dotnet/csharp/) to take advantage of Azure SQL. Azure SQL as many features for developers and learning how to take advantage of it will help you to create secure, scalable and performant modern applications. To learn more about several of the features that Azure SQL provides to developers, read here: [10 reasons to use Azure SQL in your next project](https://devblogs.microsoft.com/azure-sql/10-reasons-to-use-azure-sql-in-your-next-project/).
+This repository will guide you through the creation of a simple solution using [C#](https://docs.microsoft.com/en-us/dotnet/csharp/) to take advantage of Azure SQL. Azure SQL as many features for developers and learning how to take advantage of it will help you to create secure, scalable and performant modern applications. To learn more about several of the features that Azure SQL provides to developers, read here: [10 reasons to use Azure SQL in your next project](https://devblogs.microsoft.com/azure-sql/10-reasons-to-use-azure-sql-in-your-next-project/).

```

Comment:
Do not link to specific locales (MS Docs guideline):

```suggestion
This repository will guide you through the creation of a simple solution using [C#](https://docs.microsoft.com/dotnet/csharp/) to take advantage of Azure SQL. Azure SQL has many features for developers and learning how to take advantage of it will help you to create secure, scalable and performant modern applications. To learn more about the features that Azure SQL provides to developers, read: [10 reasons to use Azure SQL in your next project](https://devblogs.microsoft.com/azure-sql/10-reasons-to-use-azure-sql-in-your-next-project/).

```